src/components/Photo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import 'normalize.css';
+
+function Photo(){
```

Comment:
`Photo` has to take a `photo` or `city` prop, depending on what you end up naming it. 
```suggestion
function Photo({ photo }){
```